scss/civicrm/campaign/_campaign.scss

```diff
@@ -0,0 +1,43 @@
+.page-civicrm-campaign {
+  .crm-content-block {
+    @include box-shadow($box-shadow);
```

Comment:
why just for this page?

Reply:
We generally do not give shadow to this section.
This page is exception having tab element as main and other sections like table inside that tab content. so, we are adding shadow to this page and remove it from inner elements.

Reply:
can't we make it generic then, for any page with the "tab element as main"?

Reply:
No actually, these elements are not having any such containers which we can target.
In this page as well see `.crm-content-block`, this is very generic class which in major case should not have any shadow.

So, I targeted it specific to this page.

Reply:
why having a container is important? wouldn't it be enough to see, for example, if a tab element is a direct descendant of a given element?

Reply:
Seems good idea. I'ill work out on this and will let you know.